docs/utils.rst

```diff
@@ -0,0 +1,39 @@
+bayesmix/utils
```

Comment:
this page is coming out pretty weird at https://bayesmix.readthedocs.io/en/latest/utils.html 

It seems like all the docs at the head of each file get merged together

Reply:
I know right. I really don't know what's going on. I even tried to remove the RNG class from this file, or use the `doxygenclass` directive, and it still won't work.